wp-content/themes/core/blocks/tribe/site-search/block.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
```

Comment:
How useable is this block in other areas, like on pages, etc.? I'm wondering if this should be limited to being used in the masthead or not.